buildspec.yml

```diff
@@ -0,0 +1,25 @@
+version: 0.2
+
+phases:
+  install:
+    commands:
+      - sudo apt-get update
+      - sudo apt-get install apt-transport-https
+      - curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add -
+      - echo "deb https://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list
+      - sudo apt-get update && sudo apt-get install yarn
+  pre_build:
+    commands:
+      - yarn install
+  build:
+    commands:
+      - yarn build:all
+      - ./node_modules/mocha/bin/mocha -r ts-node/register test/**/*.spec.ts
+artifacts:
+  files:
+    - 'handler.js'
+  discard-paths: yes
+  base-directory: src/handlers
+cache:
+  paths:
+    - '/root/.npm/**/*'
```

Comment:
Another area that could be cached to speed up this process (especially as the project gets bigger) is to cache the `/node_modules/` directory.  Subsequent `yarn install` runs should install only new dependencies, if there are any.

Reply:
That's a good idea. I thought of this but I wasn't sure if it was the way to go, since we don't include node_modules in our git repository. But, I suppose after an initial build, node_modules will be populated, and then it can be cached from CodeBuild